src/util/psbt/map/output.rs

```diff
@@ -93,6 +93,12 @@ impl PartialEq for TapTree {
 impl Eq for TapTree {}
 
 impl TapTree {
+    /// Convenience method to retrieve number of script leafs in the tree.
+    #[inline]
+    pub fn scripts_count(&self) -> usize {
```

Comment:
As mentioned elsewhere, this should implement `ExactLenIterator` instead

Reply:
Sorry, this happens in #924: https://github.com/rust-bitcoin/rust-bitcoin/pull/924/commits/b5470bf568b93d124e2cadfaddc7bbe651d5304e

Reply:
`leaf_count` may be clearer name?

Reply:
It is not leaf count: in the future with hidden parts of the tree we will not know about how many leaves are hidden. So probably better to have it as `known_scripts_count`.